x-pack/plugin/ccr/qa/multi-cluster-with-non-compliant-license/src/test/java/org/elasticsearch/xpack/ccr/CcrMultiClusterLicenseIT.java

```diff
@@ -35,32 +30,14 @@ public void testAutoFollow() throws Exception {
         if ("follow".equals(targetCluster)) {
             final Request request = new Request("PUT", "/_ccr/auto_follow/test_pattern");
             request.setJsonEntity("{\"leader_index_patterns\":[\"*\"], \"remote_cluster\": \"leader_cluster\"}");
-            client().performRequest(request);
-
-            // parse the logs and ensure that the auto-coordinator skipped coordination on the leader cluster
-            assertBusy(() -> {
```

Comment:
I think we still might want a test for this, for the case when the license expires upstream.

Reply:
Yes, that makes sense. I just wonder how we can test this. We need to start a cluster with a platinum license and then downgrade to a basic license? But that would require another qa module, right?

Reply:
Now I think more about it, a qa module that downgrades the remote cluster, is also useful to test what happens with an index being followed.